source/frontend/package.json

```diff
@@ -56,6 +56,8 @@
     "react-router-dom": "5.2.0",
     "react-use": "^17.4.0",
     "sass": "1.38.0",
+    "stream": "0.0.2",
```

Comment:
This can be removed. I noticed it wasn't being used and removed it a few weeks ago so it can go.